scripts/fcade-quark.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
```

Comment:
The reason I did this rather than symlinking to `/app/extra/Fightcade/emulator/fcade` is because I'm not sure if there will be calls I need to stub out. Originally I wrote an early exit for `fcade://autoupdate` but that proved to not be an issue since it's read-only anyway, but I'd prefer to keep this here in case future calls to `fcade` need some middle-man translation.